src/components/Layout/Header/NavBar/MainNavLink.tsx

```diff
@@ -51,11 +51,12 @@ export const MainNavLink = memo((props: SidebarLinkProps) => {
       if (props.to) {
         e.preventDefault()
         startNavTransition(() => {
-          history.push(props.to as string)
+          // Replace paths with segments (e.g /wallet/*) to paths without (e.g /wallet)
```

Comment:
Pretty much what it says on the box, else would push to e.g `/wallet/*` which isn't a valid route to ever be in the history/url bar, this is a react-router internal

Reply:
Thankfully isolated, as this would be absolute hell otherwise to handle across the app